pkg/install/3-setuprouter.go

```diff
@@ -0,0 +1,66 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/password"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+func (i *Installer) setupRouter(ctx context.Context) error {
+	restConfig, err := restconfig.RestConfig(ctx, i.env, i.doc.OpenShiftCluster)
+	if err != nil {
+		return err
+	}
+
+	var routerIP string
+	{
+		cli, err := kubernetes.NewForConfig(restConfig)
+		if err != nil {
+			return err
+		}
+
+		svc, err := cli.CoreV1().Services("openshift-ingress").Get("router-default", metav1.GetOptions{})
```

Comment:
Needs changing. This will likely fail because it now happens before the step where we wait for clusterversion.